Library/Homebrew/dev-cmd/release-notes.rb

```diff
@@ -29,6 +33,15 @@ def release_notes
     args = release_notes_args.parse
 
     previous_tag = args.named.first
+
+    if previous_tag.present?
+
+      previous_tag_date = Date.parse Utils.popen_read(
+        "git", "-C", HOMEBREW_REPOSITORY, "log", "-1", "--format=%aI", previous_tag.sub(/\d+$/, "0")
```

Comment:
`previous_tag.sub(/\d+$/, "0")`

might be nice to split this into a separate variable so it's more obvious what it's doing.